frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -34,23 +100,60 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           title="A bucket will be created to provide the OCS Service."
           isInline
         />
-        <p className="co-legend">
+        <p className="co-legend" data-test-id="warning">
           Select at least 3 nodes in different failure domains with minimum requirements of 16 CPUs
           and 64 GiB of RAM per node.
         </p>
         <p>
           3 selected nodes are used for initial deployment. The remaining selected nodes will be
           used by OpenShift as scheduling targets for OCS scaling.
         </p>
-        <ListPage kind={NodeModel.kind} showTitle={false} ListComponent={ListComponent} />
+        <div className="ceph-node-list__max-height">
+          <ListPage
+            kind={NodeModel.kind}
+            showTitle={false}
+            ListComponent={NodeList}
+            customData={{ selectedNodes, setSelectedNodes, visibleRows, setVisibleRows }}
+          />
+        </div>
+        <p className="control-label help-block" data-test-id="nodes-selected">
+          {`${pluralize(selectedNodes?.length || 0, 'node', 'nodes')} selected`}
+        </p>
+        <div className="ceph-ocs-install__ocs-service-capacity--dropdown">
+          <OCSStorageClassDropdown onChange={setStorageClass} data-test-id="ocs-dropdown" />
+        </div>
+        <div className="ceph-ocs-install__ocs-service-capacity">
+          <label className="control-label" htmlFor="ocs-service-stoargeclass">
+            OCS Service Capacity
```

Comment:
All labels should be of same style.

![Screenshot from 2020-04-15 14-19-31](https://user-images.githubusercontent.com/25664409/79318088-33090200-7f24-11ea-92ca-34038da60144.png)
![Screenshot from 2020-04-15 14-19-41](https://user-images.githubusercontent.com/25664409/79318097-343a2f00-7f24-11ea-90cd-c23da332bc6c.png)

Reply:
`Select Nodes` is defined as title IDK if that needs change. For now, I don't want to touch it.

Reply:
@afreen23 that design does not have OCS Service Capacity.

Reply:
And helper text are not matching. `          Selected nodes will be labeled with
          <code>cluster.ocs.openshift.io/openshift-storage=&quot;&quot;</code> to create the OCS
          Service`
This one is missing from the design